public/about-us.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="MDW website">
+    <meta name="author" content="Tim Lustberg | Andre Dekker | Johan van Soest">
```

Comment:
Probably a line from previous commit, since I don't see your name in it :)

Reply:
Oops :)